app/js/features/network/saga.js

```diff
@@ -14,6 +15,7 @@ export function *doInit() {
     yield call(onReady);
     const networkId = yield call(web3.eth.net.getId);
     yield put({type: INIT_SUCCEEDED, networkId});
+    yield put({type: PRICE_INTERVAL});
```

Comment:
I'd rename the constant to `FETCH_PRICES` because it looks like the constant contains the amount of time in the interval

Reply:
is `FETCH_EXCHANGE_RATE` ok?

Reply:
Yeah